recipes/libpqxx/all/conanfile.py

```diff
@@ -0,0 +1,134 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment, CMake
+from conans.errors import ConanInvalidConfiguration
+from conans.tools import Version
+import os
+
+
+class LibpqxxRecipe(ConanFile):
+    name = "libpqxx"
+    settings = "os", "compiler", "build_type", "arch"
+    description = "The official C++ client API for PostgreSQL"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/jtv/libpqxx"
+    license = "BSD-3-Clause"
+    topics = ("conan", "libpqxx", "postgres", "postgresql", "database", "db")
+    generators = "cmake"
+    exports_sources = ["CMakeLists.txt", "patches/*"]
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    requires = "libpq/11.5"
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    @property
+    def _using_cmake(self):
+        return self.settings.os == "Windows"
```

Comment:
I assume the cmake scripts is only used for Visual Studio.
So shouldn't this be `return self.settings.compiler == "Visual Studio"`?
Because mingw.

Reply:
Windows was building with CMake before I contributed to the old repository.
I kept using CMake on Windows because MinGW requires additional steps: https://github.com/jtv/libpqxx/blob/7.0.1/INSTALL-Windows.md#mingw

In any case, use CMake on all operating systems: https://github.com/conan-io/conan-center-index/pull/887#discussion_r380180638

Reply:
I think their documentation is outdated.
I just cloned their repo and, using grep and find could not find any `MinGW.mak` file.\
(apart from NEWS and Install-Windows.md)

Reply:
It was removed between versions 6.4.5 and 7.0.0.
There were so many changes that I didn't notice it.

Reply:
cmake to rule them all :smile: